src/MetaModels/Attribute/Longtext/Longtext.php

```diff
@@ -38,6 +41,22 @@ public function getSQLDataType()
     }
 
     /**
+     * @{inheritDoc}
+     *
+     * add a fulltext-index to column after creation
+     */
+    public function createColumn()
+    {
+        parent::createColumn();
+
+        TableManipulation::addIndex(
```

Comment:
no, the Database update the index on drop/alter querys automatically.

On 03.03.2016 09:01, Christian Schiffler wrote:
> > In src/MetaModels/Attribute/Longtext/Longtext.php
<https://github.com/MetaModels/attribute_longtext/pull/12#discussion_r54846508>:
> > > @@ -38,6 +41,22 @@ public function getSQLDataType() > >      } >
>  > >      /** > > +     * @{inheritDoc} > > +     * > > +     * add a
fulltext-index to column after creation > > +     */ > > +    public
function createColumn() > > +    { > > +        parent::createColumn();
> > + > > +        TableManipulation::addIndex( > > Do we have to drop
this index prior deleting the column? > If so, we will need to override
the deleteColumn() as well. > > — > Reply to this email directly or view
it on GitHub
<https://github.com/MetaModels/attribute_longtext/pull/12/files#r54846508>.
>

-- 
--
Henry